.github/workflows/.grenrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "dataSource": "milestones",
+  "prefix": "",
+  "groupBy": {
+    ":fire: Enhancements:": ["kind/feature"],
+    ":bug: Bug fixes:": ["kind/bug"]
+  },
+  "milestoneMatch": "v{{tag_name}}",
```

Comment:
Since our tags are w/out the v prefix, should we make the milestones with the same name?

Reply:
True, fixed.